ribbon-loadbalancer/src/main/java/com/netflix/loadbalancer/LoadBalancerContext.java

```diff
@@ -324,9 +339,9 @@ public void noteOpenConnection(ServerStats serverStats) {
         }
         try {
             serverStats.incrementActiveRequestsCount();
-        } catch (Throwable e) {
-            logger.info("Unable to note Server Stats:", e);
-        }
+        } catch (Error ex) {
```

Comment:
You're right.  Don't know why I thought catching Error would be any better.  Will fix this.